src/App.js

```diff
@@ -1,24 +1,37 @@
-import logo from './logo.svg';
+/* eslint-disable import/no-extraneous-dependencies */
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { Provider } from 'react-redux';
+// import store from './redux/configStore';
+import store from './store';
+import DashboardLayout from './layouts/dashboardLayout';
+import Login from './pages/Login';
+import Register from './pages/Register';
+// import User from './pages/User';
+import Home from './pages/home';
+import Booking from './components/booking';
+import Hotels from './components/hotels';
+import HotelShow from './components/hotelShow';
 import './App.css';
+import NotFound from './components/notfound';
 
 function App() {
```

Comment:
- [ ] Good job adding the "Add hotel" feature, however, I couldn't find a way to delete rooms and/or hotels.  According to the project requirements, this is a required feature:

![image](https://user-images.githubusercontent.com/57421823/215270480-7a5659b1-a292-49de-8e56-8b26439a4c72.png)

Kindly add the feature, so we can meet the project requirements.  If due to time constraints, or any unforeseeable reason you won't be able to add it, kindly include the proper explanation in your PR description.